models/app.models.js

```diff
@@ -26,3 +26,17 @@ exports.getArticlesFromDatabase = (article_id) => {
     else return rows;
   });
 };
+
+exports.getArticlesFromDatabase = () => {
+  let query = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count
```

Comment:
I would always recommend defaulting to a const unless you are planning on re-assigning this variable, which currently doesnt seem to be the case (even if this may be the plan in the future it should only be updated when needed)